egs/librispeech/s5/local/data_prep.sh

```diff
@@ -2,11 +2,19 @@
 
 # Copyright 2014  Vassil Panayotov
 #           2014  Johns Hopkins University (author: Daniel Povey)
+#           2021  Xuechen LIU
 # Apache 2.0
 
+no_text=false
+
+. ./utils/parse_options.sh
+
 if [ "$#" -ne 2 ]; then
   echo "Usage: $0 <src-dir> <dst-dir>"
   echo "e.g.: $0 /export/a15/vpanayotov/data/LibriSpeech/dev-clean data/dev-clean"
+  echo "Options:"
+  echo "   --no-text <true|false>           # Decide to disregard text or not."
```

Comment:
Could you remove the negation, and call the option prepare_text, default true?

Reply:
changes made and tested on the 3 partitions.